README.md

```diff
@@ -1,10 +1,12 @@
 # pyedf
 
-Python package to read from and write EEG data to European Data Format files.
+pyedf is a python package to read from and write EEG data to European Data Format files.
 
-The EDF standard was introduced in [1992](https://doi.org/10.1016/0013-4694(92)90009-7). The extension of EDF with annotations was first described in [1998](https://doi.org/10.1016/S0013-4694(98)00029-7) and more formalized with the EDF+ standard that was published in [2003](https://doi.org/10.1016/S1388-2457(03)00123-8). The [EDF](http://www.edfplus.info) website describes both standards and discusses implementation details.
+Since EDF is such a widely used format, there exist multiple Python implementations for reading and writing EDF files. However, most of these Python modules consist of wrappers around the C-code implementation `edflib`, which makes installation more cumbersome and reduces portability. This implementation is in pure python with limited dependencies on external packages while having support for Python 2.7 and 3.x.
+
+The edf file, included for testing, is taken from [EDFBrowser Test Files](https://www.teuniz.net/edf_bdf_testfiles/index.html).
 
-Since EDF is such a widely used format, there exist multiple implementations for reading and writing EDF files. Most Python implementations consist of wrappers around the C-code implementation, which makes installation more cumbersome and reduces portability. This implementation is in pure Python with limited dependencies on external packages.
+The EDF standard was introduced in [1992](https://doi.org/10.1016/0013-4694(92)90009-7). The extension of EDF with annotations was first described in [1998](https://doi.org/10.1016/S0013-4694(98)00029-7) and more formalized with the EDF+ standard that was published in [2003](https://doi.org/10.1016/S1388-2457(03)00123-8). The [EDF](http://www.edfplus.info) website describes both standards and discusses implementation details.
```

Comment:
Why change this?

Reply:
It's a simple order matter. The purpose of the library is put ahead of information about EDF, as we may safely assume that people looking for edf libraries already know about edf.

Reply:
That assumption holds less often than you might otherwise think (as I know from my other job -- people look for EDF libraries because they don't know anything about EDF but suddenly have to work with it). But as a guiding principle, don't change things unnecessarily: https://yihui.org/en/2018/11/cosmetic-changes/

Reply:
Editing done. Please review.